apps/lfx-one/src/app/shared/components/changelog-drawer/changelog-drawer.component.scss

```
@@ -0,0 +1,56 @@
+// Copyright The Linux Foundation and each contributor to LFX.
+// SPDX-License-Identifier: MIT
+
+// Theme the embedded <lfx-changelog> web component to match the drawer.
+// The widget exposes CSS custom properties via :host AND ::part() selectors
+// (see @linuxfoundation/lfx-ui-core/dist/components/changelog/changelog.style.js).
+//
```

Stylelint (17.10.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)